velox/serializers/PrestoSerializer.cpp

```diff
@@ -2836,11 +2860,17 @@ void estimateFlatSerializedSize(
     for (int32_t i = 0; i < ranges.size(); ++i) {
       auto end = ranges[i].begin + ranges[i].size;
       auto numValues = bits::countBits(rawNulls, ranges[i].begin, end);
-      *(sizes[i]) +=
-          numValues * valueSize + bits::nbytes(ranges[i].size - numValues);
+      // Add the size of the values.
```

Comment:
do these fixes to existing bugs translate to updates in any unit tests? If not, wonder if we dont have coverage for these cases in the first place OR we dont bother because these are estimates